spec/unit/filter_spec.rb

```diff
@@ -177,9 +177,9 @@
   describe "#matches?" do
     let(:filter) {
       Filter.new([
-        "/unmanaged_files/name=/home/alfred",
-        "/unmanaged_files/name=/var/cache",
-        "/changed_managed_files/changes=md5,size"
+          "/unmanaged_files/files/name=/home/alfred",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.